### ISSUE TITLE
[Bug] Failed to Inject Init

### ISSUE THREAD

**Issue by mskonovalov** [Dr-TSNG/ZygiskOnKernelSU#161]
### Steps to reproduce/复现步骤

1. Disable Zigysk
2. Disable all modules
3. Reboot
4. Install Zigysk New
5. Reboot
6. See the error

### Expected behaviour/预期行为

Should work

### Actual behaviour / 实际行为

Failed to Inject Init

### Module List / 模块列表

```Shell
All disabled

OverlayFs
Play Integrity Fix
Shamiko
Systemless hosts
Zygisk detach
```

### Zygisk Next version / Zygisk Next 版本

1.2.x

### Android version / Android 版本

13

### Magisk/KernelSU/APatch version / Magisk/KernelSU/APatch 版本

magisk 28.0

### Version requirement / 版本要求

- [X] I am using latest version of Zygisk Next/我正在使用最新版本 Zygisk Next

### Logs/日志
[bootlog-2024-11-13-21-10-36.tar.gz](https://github.com/user-attachments/files/17730560/bootlog-2024-11-13-21-10-36.tar.gz)

**Comment by 5ec1cff** [comment]
DIDN'T U SEE DO NOT SEND MAGISK LOGS???

![image](https://github.com/user-attachments/assets/08765b3f-5e2a-42da-8fbc-edf2948691dc)

**Comment by mskonovalov** [comment]
Sorry, didn't notice that.
So should this work with Magisk or not?

**Comment by Astoritin** [comment]
Zygisk Next works fine in Magisk.
(If you have read the changelogs in recent versions of Zygisk Next you will find it easily)

My device runs on Android 11 / 14, Magisk Alpha f4502f8b (28001),
Zygisk Next 1.2.3(441), it works fine here.

If possible, please flash this module https://t.me/hwrtools/10 and reproduce the issue again, then reupload logs here.

**Comment by mskonovalov** [comment]
Sorry, my fault. Didn't notice it.
Just uploaded the log from LogCatcher module

**Comment by mskonovalov** [comment]
seems the error is
`real_path /proc/1/exe failed with 2: No such file or directory`

**Comment by 5ec1cff** [comment]
Will be fixed in 1.2.4
